### PR TITLE
Add the support to display one item in PlusMenu

### PR DESCRIPTION
**Pull request by hungnguyenswimlane**
## Summary

Add the support to display one item in PlusMenu.

image

## Checklist

- [x] \*Added unit tests
- [x] \*Added a code reviewer
- [x] Added changes to `/projects/swimlane/ngx-ui/CHANGELOG.md` under HEAD (Unreleased)
- [x] Updated the demo page
- [x] Included screenshots of visual changes

_\*required_

### CODE REVIEW

File: projects/swimlane/ngx-ui/src/lib/components/plus-menu/plus-menu.component.html [hunk 1]
```diff
@@ -22,7 +22,6 @@
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
   >
-
```
Comment (krattet) [inline projects/swimlane/ngx-ui/src/lib/components/plus-menu/plus-menu.component.html:25]: Wanted to double check this change was intended?

File: projects/swimlane/ngx-ui/src/lib/components/plus-menu/plus-menu.component.spec.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ const expectHtmlEquals = (actual: string, expected: string, options?: any) => {
   expect(actual).toEqual(expected, 'Expected HTML to match');
 };
 
-describe('PlusMenuComponent', () => {
+fdescribe('PlusMenuComponent', () => {
```
Comment (krattet) [inline projects/swimlane/ngx-ui/src/lib/components/plus-menu/plus-menu.component.spec.ts:22]: Reminder to remove the focused test

File: projects/swimlane/ngx-ui/src/lib/components/plus-menu/plus-menu.component.html (hunk 1 above)
Comment (hungnguyenswimlane) [reply projects/swimlane/ngx-ui/src/lib/components/plus-menu/plus-menu.component.html:25]: oh, it is not, probably a mis-enter when I look around.

**Comment by MarkColeman1** [comment]
I am going to close this PR. There is no point in displaying a plus menu component to the user with only one selectable option. You would be forcing the user into a menu when it should be a single click option. If the need to remove a menu item is needed without the `ngx-plus-menu` component causing a runtime error, use CSS to remove the menu item.

**Pull request closed without merging**